src/amenities/amenities.service.ts

```diff
@@ -21,16 +19,13 @@ export class AmenitiesService {
       const columns: Array<{ column_name: string }> = await this.prisma
         .$queryRaw`SELECT column_name FROM information_schema.columns WHERE table_name = 'Amenity';`;
       const excludedColumns = ['accommodationId', 'id'];
-      const list = columns
+      const amenitiesList = columns
```

Comment:
here is possible to make all transformation an filtering within one loop, but it makes a not huge impact for such small amount of elements